### ISSUE TITLE
Kill command and Bestial Wrath

### ISSUE THREAD

**Issue by Tritonek** [dalaranwow/dalaran-wow#5010]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
After using "Kill Command" and "Bestial Wrath", the bonus damage doesn't affect the ability "Bite".
**Expected behaviour**: Tell us what should happen instead
After using either of those abilities (Kill Command or Bestial Wrath) the damage of the pets auto attacks and all abilities should be increased by a certain amount.
**Steps to reproduce the problem**:

1.  Check the formula you are currently using for Claw.
2.  Use the same formula for Bite.
3.  /shrug

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**
Kill Command: [spell=83381]
![wowscrnshot_110417_100737](https://user-images.githubusercontent.com/18738446/32404257-dc1c07cc-c14c-11e7-8ba7-58712e2171a4.jpg)
![wowscrnshot_110417_100741](https://user-images.githubusercontent.com/18738446/32404263-ee4a5264-c14c-11e7-9f04-9d88df3bcf4a.jpg)
![wowscrnshot_110417_100845](https://user-images.githubusercontent.com/18738446/32404264-f62342ca-c14c-11e7-9d56-9e7e9e7d298e.jpg)
![wowscrnshot_110417_100849](https://user-images.githubusercontent.com/18738446/32404267-fa56b9ee-c14c-11e7-81ae-787e0ffcb651.jpg)

Bestial Wrath:  [spell=38484]
![wowscrnshot_110417_101623](https://user-images.githubusercontent.com/18738446/32404271-06ac71de-c14d-11e7-84a3-fcaa0040ed3c.jpg)
![wowscrnshot_110417_101628](https://user-images.githubusercontent.com/18738446/32404272-09f44696-c14d-11e7-9560-c6665e9fa5f6.jpg)
![wowscrnshot_110417_101812](https://user-images.githubusercontent.com/18738446/32404273-0cea3c3e-c14d-11e7-8c69-2263f03ce448.jpg)
![wowscrnshot_110417_101821](https://user-images.githubusercontent.com/18738446/32404275-106b1874-c14d-11e7-9f7d-617094c9afbf.jpg)

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Works as intended on PTR.
